test/native/jest.config.js

```diff
@@ -13,48 +13,45 @@ if ( process.env.TEST_RN_PLATFORM ) {
 	console.log( 'Setting RN platform to: default (' + defaultPlatform + ')' );
 }
 
-const configPath = 'test/native';
-
 const transpiledPackageNames = glob( '../../packages/*/src/index.{js,ts}' ).map(
 	( fileName ) => fileName.split( '/' )[ 3 ]
 );
 
 module.exports = {
 	verbose: true,
-	rootDir: '../../',
+	rootDir: '.',
+	roots: [ '<rootDir>/../..' ],
 	// Automatically clear mock calls and instances between every test.
 	clearMocks: true,
 	preset: 'react-native',
-	setupFiles: [ '<rootDir>/' + configPath + '/setup.js' ],
-	setupFilesAfterEnv: [ '<rootDir>/' + configPath + '/setup-after-env.js' ],
+	setupFiles: [ '<rootDir>/setup.js' ],
+	setupFilesAfterEnv: [ '<rootDir>/setup-after-env.js' ],
 	testMatch: [
-		'**/test/!(helper)*.native.[jt]s?(x)',
-		'<rootDir>/packages/react-native-*/**/?(*.)+(spec|test).[jt]s?(x)',
-	],
-	testPathIgnorePatterns: [
-		'/node_modules/',
-		'/__device-tests__/',
-		'<rootDir>/.*/build/',
-		'<rootDir>/.*/build-module/',
```

Comment:
Do we still need to keep these ignore patterns? Or is there a good reason we're removing them?